frontend/src/stores/hydrologic.js

```diff
@@ -271,27 +292,33 @@ export const useHydrologicStore = defineStore('hydrologic', () => {
       default: false,
       short_definition:
         'Maximum flow accumulation value for a node or reach. Flow accumulation values are extracted from the MERIT Hydro dataset (Yamazaki et al., 2019)',
-      units: 'square kilometers',
+      swotviz_alias: 'Maximum Flow Accumulation',
+        units: 'square kilometers',
       plottable: true,
-      plot_definition: 'Flow Accumulation'
+      plot_definition: 'Flow Accumulation',
+      significant_figures: 0 //TODO: Check this
     },
     {
       abbreviation: 'n_chan_max',
       definition: 'Maximum number of channels for each node or reach',
       fileType: 'all',
       default: false,
       short_definition: 'Maximum number of channels for each node or reach',
+      swotviz_alias: 'Maximum Channel Count',
       units: '',
-      plottable: false
+      plottable: false,
+      significant_figures: 0 //TODO: Check this
     },
     {
       abbreviation: 'n_chan_mod',
       definition: 'Mode of the number of channels for each node or reach',
       fileType: 'all',
       default: false,
       short_definition: 'Mode of the number of channels for each node or reach',
+      swotviz_alias: 'Mode Channel Count',
       units: '',
-      plottable: false
+      plottable: false,
+      significant_figures: 0 //TODO: Check this
```

Comment:
you have a bunch of `TODO`s here...
the sig figs look good to me.
maybe remove the Todos if no longer applicable?

Reply:
Removed TODOs.